### DISCUSSION TITLE
Feature Request: Support for Multiple Prisma Schema Files in Epic Stack

### DISCUSSION THREAD

**Discussion by MoSattler** [epicweb-dev/epic-stack#821] [Ideas]
Prisma supports multiple schema files, as outlined [here](https://www.prisma.io/docs/orm/prisma-schema/overview/location#multi-file-prisma-schema). However, this doesn't currently work with the Epic Stack, as the setup script and a few other things break when multiple schema files are enabled. It could be useful if the Epic Stack could be adjusted to make it easier for users to migrate to multiple Prisma schema files if they choose to do so.

**Comment by andrecasal** [comment]
Out of curiosity, what's the biggest benefit of multi-file schema for you?

Shortening the time it takes to find the right model to edit?
Seeing how many models you've got at a glance? If so, why is that important? What does it help you accomplish better or faster?

Genuine questions, I'm not judging :) Cheers mate 🙌

**Reply by MoSattler** [reply]
Hey @andrecasal - [this](https://www.prisma.io/blog/organize-your-prisma-schema-with-multi-file-support#when-to-use-multi-file-schemas) really describes my need for it! Cheers!

**Comment by kentcdodds** [comment]
Happy to review a PR that makes multiple schema files possible without jumping to multiple files. I think having a single file is simpler and multiple files should only be used when the number of models gets unwieldy.
